tests/integrations/django/asgi/test_asgi.py

```diff
@@ -68,3 +76,83 @@ async def test_async_views(sentry_init, capture_events, application):
         "query_string": None,
         "url": "/async_message",
     }
+
+
+@pytest.mark.parametrize("application", APPS)
+@pytest.mark.asyncio
+@pytest.mark.skipif(
+    django.VERSION < (3, 1), reason="async views have been introduced in Django 3.1"
+)
+async def test_async_views_concurrent_execution(
+    sentry_init, capture_events, application, settings
+):
+
+    import asyncio
+    import time
+
+    if application is asgi_application:
+        settings.MIDDLEWARE = []
+        asgi_application.load_middleware(is_async=True)
+
+    sentry_init(integrations=[DjangoIntegration()], send_default_pii=True)
+
+    comm = HttpCommunicator(application, "GET", "/my_async_view")
+    comm2 = HttpCommunicator(application, "GET", "/my_async_view")
+    start = time.time()
+
+    (resp1, resp2), _ = await asyncio.wait(
+        [comm.get_response(timeout=5), comm2.get_response(timeout=5)]
+    )
+
+    end = time.time()
+
+    assert resp1.result()["status"] == 200
+    assert resp2.result()["status"] == 200
+
+    assert end - start < 1.5
```

Comment:
Appears to still fail for channels, not sure why.

Reply:
Where do you mean? This last CI run shows that the tests are failing for Python 3.6, 3.7, 3.8 for django 3.1 and dev, am I missing something?

Reply:
Right but within those environments only hte channels variant is failing afaict.

Reply:
decided to skip test for channels